core/arch/arm/include/kernel/user_ta.h

```diff
@@ -87,20 +88,28 @@ struct user_ta_store_ops;
 #ifdef CFG_WITH_USER_TA
 TEE_Result tee_ta_init_user_ta_session(const TEE_UUID *uuid,
 			struct tee_ta_session *s);
-TEE_Result tee_ta_register_ta_store(struct user_ta_store_ops *ops);
 #else
 static inline TEE_Result tee_ta_init_user_ta_session(
 			const TEE_UUID *uuid __unused,
 			struct tee_ta_session *s __unused)
 {
 	return TEE_ERROR_ITEM_NOT_FOUND;
 }
-
-static inline TEE_Result tee_ta_register_ta_store(
-			struct user_ta_store_ops *ops __unused)
-{
-	return TEE_ERROR_NOT_SUPPORTED;
-}
 #endif
 
+/*
+ * Registers a TA storage.
+ *
```

Comment:
late comment: should describe the priority level.
Was described as:
```
Higher priority instances are tried first when a TA is looked up.	
Priority lower value means higher priority.
```
Maybe shorter `TA storage of lower prio value as lookup first`.
Low

Reply:
You're right, we should describe the priority level. I'll add something along that.